_posts/2024-04-07-WildFly32Beta-released.adoc

```diff
@@ -0,0 +1,164 @@
+
+=== Tooling Support for Feature Stability Levels
+
+Our Galleon-based provisioning tooling has also had updates related to feature stability levels: we've added configuration options to allow you to control the stability level of features in your installation. This can be used to do things like:
+
+* Prevent the provisioning of lower stability features, so they are not available for use even when the `--stability` server start param is used.
+* Enable the inclusion of lower stability features in the configuration files the provisioning tool generates, avoiding the need to use a post-provisioning tool like the WildFly CLI to incorporate them into the configuration.
+
+To limit your installation level to the highest stability features, you would include the following in your maven plugin configuration:
+
+```
+<galleon-options>
+    <stability-level>default</stability-level>
+</galleon-options>
+```
+
+To allow Galleon to include lower stability features in your installation's generated configuration files, you could do something like:
+
+```
+<galleon-options>
+    <stability-level>preview</stability-level>
+</galleon-options>
+```
+
+[NOTE]
+====
+If one wants to have different values for configuration files and packages (i.e. filesystem resources like JBoss Modules modules), then the `<config-stability-level>` and `<package-stability-level>` options are to be used instead of `<stability-level>`.
+The use case for using `config-stability-level` and `package-stability-level` as an alternative to `stability-level` 
+is when the user wishes to generates configurations with features at a given stability level 
+while allowing provisioning of packages at a lower level. 
+The presence of the lower stability level packages allows subsequent update of the configuration, e.g. with the WildFly CLI, to enable lower stability features.
+====
+
+The link:https://docs.wildfly.org/wildfly-maven-plugin/releases/5.0/[wildfly-maven-plugin 5.0.0.Beta5], link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/releases/tag/11.0.0.Beta2[wildfly-jar-maven-plugin 11.0.0.Beta2] (for bootable jars) and the link:https://github.com/wildfly/galleon/releases/tag/6.0.0.Beta6[Galleon 6.0.0.Beta6] tools all support these stability level configuration options. I encourage you to try them out.
+
+== WildFly Glow
+
+We've also continued to make improvements in link:https://www.wildfly.org/news/2024/01/29/wildfly-glow/[WildFly Glow]. Please try to out the link:https://github.com/wildfly/wildfly-glow/releases/tag/1.0.0.Beta13[1.0.0.Beta13 release] and give us your feedback as we head toward the finish line and WildFly Glow 1.0.0.Final.
```

Comment:
Please try to out the ==> Please try out the

Reply:
Done, thank you